syntacticAnalysis.py

```diff
@@ -81,6 +75,17 @@ def syntacticAnalysis(nlp, line):
         if attr_pos == 0:
             keywords.append((getPhrase(question, attr_pos), "question_word"))
 
+    elif root_pos == 0 or aux_pos == 0:
+        # Likely a yes/no question
+        if aux_pos == 0:
+            keywords.append((getPhrase(question, aux_pos), "question_word"))
+            keywords.append((getPhrase(question, root_pos), "property"))
+            keywords.append((getPhrase(question, pobj_pos), "property_attribute"))
```

Comment:
Can you clarify what a property_attribute is?

Reply:
It's an attribute that has to exist for the given property - Did X do Y with Z? Is Z Y of X? Z here is  the property attribute, Y - the attribute

Reply:
Could you maybe at this in comments?